DiscordBot.Domain/Database/DatabaseContainer.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using DiscordBot.Domain.Configuration;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.Options;
+
+namespace DiscordBot.Domain.Database
```

Comment:
This is in a complete wrong namespace

Reply:
Should I use DiscordBot.Domain.Abstractions?

Reply:
Code doesnt belong into the Domain namespace. It all should be moved to something like DiscordBot.DataAccessLayer or DiscordBot.Dal